### ISSUE TITLE
Errors in order summary without log in

### ISSUE THREAD

**Issue by diegomassaroo** [scottyzen/woonuxt#212]
When I complete an order it logs me this error:
TypeError: Cannot read properties of undefined (reading 'replace')
If I check the props.status in orderStatusLabel it shows me empty receipt:
<img width="789" alt="Screenshot 2024-08-02 at 12 29 11" src="https://github.com/user-attachments/assets/6fa0952c-61b3-4018-8c39-eb22a652164b">

Then when I try to change page and navigate the website it logs me this error:
Cannot destructure property 'type' of 'vnode' as it is null.

Do you have any suggestions?

**Comment by alexookah** [comment]
@diegomassaroo 
Are you using latest version ?

There was a similar issue:
https://github.com/scottyzen/woonuxt/issues/206
Try using latest version maybe it's related/similar issue and your problem will be resolved.

**Comment by diegomassaroo** [comment]
I have tried but it doesn't resolve the issue

I am using:
WordPress | 6.6.1
WooCommerce | 9.1.4
WPGraphQL | 1.27.2
WooGraphQL | 0.19.0
WPGraphQL CORS | 2.1
Woonuxt | 1.0.56

**Comment by alexookah** [comment]
it seems that the order.status is missing. Please try the latest WooGraphQL | 0.20.0

**Comment by alexookah** [comment]
@diegomassaroo 
it seems that the order.status is missing. Please try the latest WooGraphQL | 0.20.0

**Comment by alexookah** [comment]
Let us know if this resolved your problem and if resolved close this issue :)

**Comment by diegomassaroo** [comment]
I update WooGraphQL but it doesn't resolve the problem

**Comment by alexookah** [comment]
What payment do you use to checkout? If you go to the order list does it show?

**Comment by diegomassaroo** [comment]
I use stripe card and yes they appear

**Comment by alexookah** [comment]
What status do they have? how did you make the order?

**Comment by alexookah** [comment]
There is a pending [PR ](https://github.com/scottyzen/woonuxt/pull/207)which Improves checkout with stripe. And this might happen in orders when the payment has failed or some error. so the status is not defined.
But this requires a bit of testing.

@scottyzen 
Will you check the PR and maybe create a new version for these issues to get fixed?

**Comment by diegomassaroo** [comment]
> What status do they have?

They have in processing status, so they are correct because the product aren't virtual or downloadable.

**Comment by alexookah** [comment]
@diegomassaroo 
with the last version did the issue got resolved?

**Comment by diegomassaroo** [comment]
In chrome it works but it display an empty receipt. Instead in incognito and on safari it display: not authorized to access this order

**Comment by alexookah** [comment]
And this is only happening on guest checkout right?

**Comment by diegomassaroo** [comment]
Yes exactly

**Comment by alexookah** [comment]
@diegomassaroo 
I see @scottyzen updated some code about this issue. Could you check the latest version and see if the problem is resolved now?

**Comment by diegomassaroo** [comment]
I just checked and it still remains in a 'Logging you in...' status

**Comment by alexookah** [comment]
I just made a guest order and it worked. Are you sure you are testing towards latest code versions? (woonuxt, wp-graphQL, woocommerce) ?

**Comment by diegomassaroo** [comment]
Yes, all my plugins are updated, but I still encounter this issue when completing a guest order:
<img width="792" alt="Screenshot 2024-08-31 at 15 08 49" src="https://github.com/user-attachments/assets/5bac18e1-e84e-4030-81a8-62f931422f30">

**Comment by alexookah** [comment]
What option do you have in WP-GRAPHQL-CORS? do you have set the `SameSite CookieMode` to `None`?

**Comment by alexookah** [comment]
also how is it in safari and how is it in Chrome?

**Comment by alexookah** [comment]
Are you using the latest main code? can you share your website ?

**Comment by diegomassaroo** [comment]
Sure, here’s the screenshot from WP-GraphQL-CORS: 
<img width="1082" alt="Screenshot 2024-09-01 at 01 13 42" src="https://github.com/user-attachments/assets/540fe0bd-1d03-4a6a-94e1-183816f66d41">

Yes, I’ve tested it on both Safari and Chrome. 

This is the link to my website on Vercel:
https://bikeinnovation-fe.vercel.app

As I mentioned last time, I also noticed that when I try to log in, it remains stuck on the 'Logging you in...' status.

**Comment by alexookah** [comment]
I just checked your website and I just found out you are using outdated code from main branch. 
Please try to rebase from main your website to get latest changes and try again. it should work.

**Comment by diegomassaroo** [comment]
I’ve downloaded a copy of the latest code and updated it with my GraphQL endpoint, but it still doesn’t seem to be working properly.

**Comment by alexookah** [comment]
What's your graphql endpoint?

**Comment by alexookah** [comment]
Also I am not sure if that vercel should work. Try connecting it with another subdomain same as the Wordpress. Maybe that's the issue.

**Comment by alexookah** [comment]
Also can you successfully login in localhost?

**Comment by diegomassaroo** [comment]
Yes, I’ve tried reducing the authorized domain, but I still can’t log in, even on localhost.

**Comment by alexookah** [comment]
I encountered the same issue on my Woonuxt website when testing in Safari — getting stuck on the `"Logging you in..." `message.

It seems Safari has stricter security measures, particularly when connecting to an HTTPS website from localhost without SSL, which might cause cookies to be ignored.

However, my website works correctly when accessed via HTTPS without a subdomain: https://comewithreverse.com/. My WordPress site is located at: https://shop.comewithreverse.com/.

Please try setting up a subdomain at https://staging-frontend.bikeinnovation.it/, connect it with Vercel, and test again in Safari. This should resolve the issue.

**Comment by diegomassaroo** [comment]
I tried with another site that’s already published, which shares the same domain as my WordPress site, and I’m getting the same error.

https://staging.anienepublishing.it WordPress site
https://anienepublishing.it Published site

**Comment by alexookah** [comment]
Can you share that domain you tested?

**Comment by diegomassaroo** [comment]
Sure, but I’ll need to enable test mode for Stripe if you want to test it. Just let me know, and I’ll activate it. 
Here’s the link: https://www.anienepublishing.it/

**Comment by alexookah** [comment]
I tested in the website login with safari and it worked I didnt get any "Logging you in...".
the wordpress is in url: https://staging.anienepublishing.it/graphql. So not the same issue.

So maybe in checkout there is something else.

 I remember when I worked on this [PR](https://github.com/scottyzen/woonuxt/pull/207) even if there is a checkout payment error the order was created. How did you test exactly the checkout in the aniene website?

**Comment by diegomassaroo** [comment]
Yes, Aniene is using the repository from August 27, 2024, but I haven’t used layers because I didn’t succeed last time. The locale translations didn’t work as expected, so next time I’ll use layers from the beginning. I select one or more products, fill in the billing details, and proceed to payment. The order completes, but I’m still seeing that error screen.

**Comment by alexookah** [comment]
Can you Add me in Discord, maybe we can sync so that I can test it in test mode?

**Comment by Karakrisz** [comment]
Here, we are dealing with a live website, and it doesn’t allow login unless I disable tracking in Safari. The mentioned website => https://www.fuggonyoazis.hu
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/b307f096-168f-45ea-88fe-bbb80c5c6a42" />
